MonkeyChat/ChatViewController.swift

```diff
@@ -229,10 +242,20 @@ class ChatViewController: JSQMessagesViewController, JSQMessagesComposerTextView
     self.outgoingBubbleImageData = bubbleFactory?.outgoingMessagesBubbleImage(with: .jsq_messageBubbleBlue())
     self.incomingBubbleImageData = bubbleFactory?.incomingMessagesBubbleImage(with: .jsq_messageBubbleLightGray())
     
-    
     self.collectionView.reloadData()
   }
   
+  override func viewWillDisappear(_ animated: Bool) {
+    super.viewWillDisappear(animated)
+    
+    if(self.audioBubbleOnPlay != nil){
```

Comment:
now that self.audioBubbleOnPlay is an optional, we can use a `guard` statement here